Docs: Navigation menu wrap-up: About, Features and Advanced Features as Reference (Diátaxis) (#10010)
readthedocs/readthedocs.org: docs/user/about/index.rst

About Read the Docs
===================

.. toctree::
   :caption: Additional commercial features

>    :hidden:

Reviewer: How are users meant to find these pages, other than the sidebar? Are they linked anywhere else?

Author: A description list would be nice to introduce the content in this section!

Author: Caption in the TOC was also wrong...

We are proud about the way we manage our company and products,
and are glad to have you on board with us in this :doc:`great documentation journey </story>`.

> A reference to our organization
> -------------------------------
> 
> As a company that works with documentation,
> we enjoy using documentation tools for maintaining most of our information.
> For instance,
> we find that Git and documentation tools are extremely efficient for maintaining policies and legal documents.

Reviewer: ```suggestion
Additional content
-------------------------------

If you want to dive more into more specific information and our policies,
we've brought most of the most important ones below.   
```

> .. seealso::

Reviewer: Why is this a seealso, but not the other links?

   `Our website <https://about.readthedocs.com>`__

>       You won't find everything explained here.
>       The most general-purpose information about Read the Docs is available on our main website.

Reviewer: ```suggestion
      Our primary website has general-purpose information about Read the Docs like pricing and feature overviews.
```

> Here is a list of information about our organization that we currently maintain using documentation tools:

Reviewer: This is really oddly specific? I think it's awkard to try and explain why we're using "docs tools" for this twice, instead of just referencing it?

```suggestion
```

Author: I agree, sorry about dumping an experiment into the PR. I was sketching how it would look if we tried to include the user into a "here is what we enjoy using documentation tools for" (such as legal documents and policies) -- but it became very meta. Could perhaps figure in an admonition box some other sunny day :)

Author: (it was a last minute commit&push before signing off Friday)
